src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/prisonperson/PrisonPersonMigrationMapping.kt

```diff
@@ -12,6 +12,8 @@ data class PrisonPersonMigrationMapping(
 
   val migrationType: PrisonPersonMigrationType,
 
+  val dpsIds: String,
+
```

Comment:
For prison person there will be several migrations (each of a different PrisonPersonMigrationType). The mappings won't be used to link DPS and NOMIS entities and are only there to help monitor the migration. The DPS IDs created could be of any type and could a single value or a collection.

For these reasons I'm just going to dump the DPS IDs into a String column rather than have to worry about their type or a child table for collections.